packages/arco-lib/src/components/Tabs.tsx

```diff
@@ -119,15 +119,21 @@ export const Tabs = implementRuntimeComponent({
       activeTab={String(activeTab)}
       ref={ref}
     >
-      {tabs.map((tabName, idx) => (
-        <TabPane key={String(idx)} title={tabName}>
-          {slotsElements?.content
-            ? slotsElements.content({
-                tabIndex: idx,
-              })
-            : null}
-        </TabPane>
-      ))}
+      {tabs.map((tabItem, idx) =>
+        tabItem.hidden ? null : (
+          <TabPane
```

Comment:
Should it be `TabPanel`?

Reply:
I'm not sure, Arco used `pane`, a word that means a piece, section or side of something, which also seems correct